docs/source/trainer.rst

```diff
@@ -632,17 +632,10 @@ fast_dev_run
 
 |
 
-.. raw:: html
-
-    <video width="50%" max-width="400px" controls
```

Comment:
why is the video gone?

Reply:
same video was being added twice https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#fast-dev-run

Reply:
it's fixed now in another PR.